src/fizzbuzz/domain/DisplaySentences.ts

```diff
@@ -0,0 +1,9 @@
+import { GetSentencesUsecase } from '../usecase/getSentences/getSentences.usecase';
+
+export abstract class DisplaySentences {
+  useCase: GetSentencesUsecase = new GetSentencesUsecase();
+
+  display() {
```

Comment:
if method does nothing should be abstract display(): void; to force its implementation

Reply:
Exactly, i forgot to clear this method